build-system/pr-check/e2e-experiment-tests.js

```diff
@@ -22,28 +22,47 @@
  */
 
 const colors = require('ansi-colors');
+const experimentsConfig = require('../global-configs/experiments-config.json');
 const {
+  downloadDistExperimentOutput,
   downloadDistOutput,
   printChangeSummary,
   startTimer,
+  startSauceConnect,
   stopTimer,
+  stopSauceConnect,
   timedExecOrDie: timedExecOrDieBase,
 } = require('./utils');
 const {determineBuildTargets} = require('./build-targets');
 const {isTravisPullRequestBuild} = require('../travis');
 
-const FILENAME = 'e2e-tests.js';
+const FILENAME = 'e2e-experiment-tests.js';
 const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
 const timedExecOrDie = (cmd, unusedFileName) =>
   timedExecOrDieBase(cmd, FILENAME);
 
+async function runExperimentTests_() {
+  await startSauceConnect(FILENAME);
+  Object.keys(experimentsConfig).forEach(experiment => {
+    const config = experimentsConfig[experiment];
+    if (config.command) {
+      timedExecOrDie('gulp clean');
+      downloadDistExperimentOutput(FILENAME, experiment);
+      timedExecOrDie('gulp update-packages');
+      timedExecOrDie('gulp integration --nobuild --compiled --saucelabs');
+    }
+  });
+  stopSauceConnect(FILENAME);
+}
+
 async function main() {
   const startTime = startTimer(FILENAME, FILENAME);
 
   if (!isTravisPullRequestBuild()) {
     downloadDistOutput(FILENAME);
     timedExecOrDie('gulp update-packages');
     timedExecOrDie('gulp e2e --nobuild --headless');
+    await runExperimentTests_();
```

Comment:
Curious: What is the new running time of the E2E job? (I'd imagine it's still shorter than or close to the local / remote tests jobs?)